core/controllers/editor.py

```diff
@@ -71,6 +71,45 @@ def _require_valid_version(version_from_payload, exploration_version):
 }
 
 
+def replace_ids_with_names_in_dict(exploration_id):
+    """Replace ids with the names and return rights as a dictionary.
+
+    Args:
+        exploration_id: str. ID of the exploration.
+
+    Returns:
+        dict. The dictionary where ids are replaced with corresponding names.
+    """
+
+    rights_dict = rights_manager.get_exploration_rights(
+        exploration_id).to_dict()
+
+    if rights_dict['community_owned']:
+        rights_dict['owner_names'] = []
+        rights_dict['editor_names'] = []
+        rights_dict['voice_artist_names'] = []
+        rights_dict['viewer_names'] = []
```

Comment:
Why is this needed here?

Reply:
Because of https://github.com/oppia/oppia/blob/fcd109d7b7d60f4324346cb6a548109372e52f63/core/domain/rights_domain.py#L161-L171
if it is community owned, it should have empty lists.

Reply:
But then we will get the empty list of IDs and do not need special exception for it here, right?

Reply:
Yeah, okay. I'll remove those.